apps/changelog/classes/change_log.py

```diff
@@ -63,7 +63,8 @@ def write_changelog(
         buf.append('')
 
         with open(changelog_file_name, 'w') as f:
-            f.write('\n'.join(buf))
+            string_to_write = '\n'.join(buf)
+            f.write(string_to_write.encode('UTF-8'))
```

Comment:
Please stylize this as `utf-8` (lowercase)

Reply:
Done